src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/CellUpdate.cs

```diff
@@ -89,7 +89,7 @@ public CellUpdate(DbColumn column, string valueAsString)
         /// <summary>
         /// The column that the cell will be placed in
         /// </summary>
-        public DbColumn Column { get; }
+        public DbColumnWrapper Column { get; }
```

Comment:
So am I 😄 
But Kevin said a while back that the convention here is to do member variables -> constructor -> properties -> public methods -> private methods.